### PR TITLE
Dtspo 178

### PR DESCRIPTION
**Pull request by zc-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###
https://tools.hmcts.net/jira/browse/DTSPO-178

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/environments/sbox/common-overlay/toffee/frontend-patch.yaml [hunk 1]
```diff
@@ -11,4 +11,11 @@ spec:
       environment:
         RECIPE_BACKEND_URL: toffee-recipe-backend.{{ .Values.global.environment }}.platform.hmcts.net
     global:
-      environment: sandbox
\ No newline at end of file
+      environment: sandbox
+    volumes:
```
Comment (timja) [inline k8s/environments/sbox/common-overlay/toffee/frontend-patch.yaml:15]: why is this needed?

**Review by chrisorisawayi — APPROVED** [review]
